debezium-core/src/main/java/io/debezium/cloudevents/CloudEventsConverterConfig.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.cloudevents;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.storage.ConverterConfig;
+
+/**
+ * Configuration options for {@link CloudEventsConverter CloudEventsConverter} instances.
+ */
+public class CloudEventsConverterConfig extends ConverterConfig {
+
+    public static final String CLOUDEVENTS_SERIALIZER_TYPE_CONFIG = "cloudevents.serializer.type";
+    public static final String CLOUDEVENTS_SERIALIZER_TYPE_DEFAULT = "json";
+    private static final String CLOUDEVENTS_SERIALIZER_TYPE_DOC = "Specify a serializer to serialize CloudEvents values";
+
+    public static final String CLOUDEVENTS_DATA_SERIALIZER_TYPE_CONFIG = "cloudevents.data.serializer.type";
+    public static final String CLOUDEVENTS_DATA_SERIALIZER_TYPE_DEFAULT = "json";
+    private static final String CLOUDEVENTS_DATA_SERIALIZER_TYPE_DOC = "Specify a serializer to serialize the data field of CloudEvents values";
+
+    public static final String CLOUDEVENTS_JSON_SCHEMAS_ENABLE_CONFIG = "json.schemas.enable";
+    public static final boolean CLOUDEVENTS_JSON_SCHEMAS_ENABLE_DEFAULT = false;
+    private static final String CLOUDEVENTS_JSON_SCHEMAS_ENABLE_DOC = "Include json schemas within each of the serialized CloudEvents data.";
+
+    public static final String CLOUDEVENTS_SCHEMA_REGISTRY_URL_CONFIG = "schema.registry.url";
+    private static final String CLOUDEVENTS_SCHEMA_REGISTRY_URL_DOC = "Comma-separated list of URLs for schema registry instances that can be used to register or look up schemas for CloudEvents.";
+
+    public static final String CLOUDEVENTS_DATA_SCHEMA_REGISTRY_URL_CONFIG = "data.schema.registry.url";
```

Comment:
I don't think a separate option is needed, it seems very unlikely that one would use different registries for the envelope and the data attribute.